### ISSUE TITLE
PageField-Choices only available on first Inline

### ISSUE THREAD

**Issue by rprader** [divio/django-cms#3904]
When using the PageField() in a model which has a ForeignKey to another model, the available Page-Choices are only available on the first row in the inline-admin.

So I have for example the following models:
```
class NewsEntry(models.Model):
    title = models.CharField(verbose_name='Titel', max_length=255)
    slug = models.SlugField(verbose_name='Slug', help_text='Dient zur Bildung der URL', unique=True)
    lead = models.TextField(verbose_name='Lead')
    date = models.DateTimeField(verbose_name='Datum')
    read_amount = models.IntegerField(verbose_name='Anzahl gelesen', default=0, editable=False)


class NewsEntryLink(models.Model):
	title = models.CharField(max_length=255, verbose_name='Titel')
    entry = models.ForeignKey(NewsEntry, verbose_name='News-Eintrag', related_name='links')
    page_link = PageField(verbose_name='Seite', blank=True, null=True, help_text=u'Hat höhere Priorität gegenüber dem externen Link.')
    external_link = models.URLField(verbose_name='Externer Link', blank=True, null=True)
```

And defined them for admin with InlineAdmin:
```
class NewsEntryLinkAdmin(admin.TabularInline):
    model = NewsEntryLink
    extra = 1

class NewsEntryAdmin(admin.ModelAdmin):
    inlines = [NewsEntryLinkAdmin, ]


admin.site.register(NewsEntry, NewsEntryAdmin)
```

With this, it's not possible to add more than one related data set, because the available pages are only listed in the first row of the inline admin.

There's also a js-error thrown when opening the add-new-item page:

```
Uncaught TypeError: undefined is not a function
```

**Comment by yakky** [comment]
Can't reproduce, neither on develop or on 3.0.10
Could you provide more information about the setup? Do you use any custom admin skin?

**Comment by rprader** [comment]
I do use a custom admin skin, namely djangocms-admin-style v0.2.5
But it doesn't make a difference if I deactivate this plugin.

Here are some other of my package-versions, I'm using:

```
django = 1.5.12
django_cms = 3.0.10
django_hvad = 1.0.0
django_mptt = 0.6.1
django-pipeline = 1.3.9
django-polymorphic = 0.6.1
```

Do you need any further information?

**Comment by yakky** [comment]
Thanks, I've been able to reproduce this in django 1.5 (likely due to outdated jquery version)

**Comment by yakky** [comment]
@rprader could you please test if #3914 fixes the error for you?

**Comment by rprader** [comment]
Yes it does, awesome! Thanks a lot!

**Comment by yakky** [comment]
Awesome, will merge shortly and it will be shipped with 3.0.11 (due in a very short timeframe)

**Comment by yakky** [comment]
Closed by #3914
